### PR TITLE
Add LTE APIs

### PR DESCRIPTION
**Pull request by DavidCurtiss**
### What does this Pull Request accomplish?

Adds LTE APIs

### Why should this Pull Request be merged?

[AB#1804659](https://ni.visualstudio.com/DevCentral/_workitems/edit/1804659)

### What testing has been done?

```
Z:\dev\grpc-device\build\debug>SystemTestsRunner.exe --gtest_filter=*RFmxL*
Note: Google Test filter = *RFmxL*
[==========] Running 7 tests from 2 test suites.
[----------] Global test environment set-up.
[----------] 1 test from NiRFmxLTEDriverApiTests
[ RUN      ] NiRFmxLTEDriverApiTests.Init_Close_Succeeds
[       OK ] NiRFmxLTEDriverApiTests.Init_Close_Succeeds (37664 ms)
[----------] 1 test from NiRFmxLTEDriverApiTests (37669 ms total)

[----------] 6 tests from NiRFmxLTESessionTest
[ RUN      ] NiRFmxLTESessionTest.InitializeSessionWithDeviceAndSessionName_CreatesDriverSession
[       OK ] NiRFmxLTESessionTest.InitializeSessionWithDeviceAndSessionName_CreatesDriverSession (909 ms)
[ RUN      ] NiRFmxLTESessionTest.InitializeSessionWithDeviceAndNoSessionName_CreatesDriverSession
[       OK ] NiRFmxLTESessionTest.InitializeSessionWithDeviceAndNoSessionName_CreatesDriverSession (1248 ms)
[ RUN      ] NiRFmxLTESessionTest.InitializeSessionWithoutDevice_ReturnsDriverError
[       OK ] NiRFmxLTESessionTest.InitializeSessionWithoutDevice_ReturnsDriverError (812 ms)
[ RUN      ] NiRFmxLTESessionTest.InitializedSession_CloseSession_ClosesDriverSession
[       OK ] NiRFmxLTESessionTest.InitializedSession_CloseSession_ClosesDriverSession (1231 ms)
[ RUN      ] NiRFmxLTESessionTest.TwoInitializedSessionsOnSameDevice_CloseSessions_ClosesDriverSessions
[       OK ] NiRFmxLTESessionTest.TwoInitializedSessionsOnSameDevice_CloseSessions_ClosesDriverSessions (1683 ms)
[ RUN      ] NiRFmxLTESessionTest.InvalidSession_CloseSession_ReturnsInvalidSessionError
[       OK ] NiRFmxLTESessionTest.InvalidSession_CloseSession_ReturnsInvalidSessionError (13 ms)
[----------] 6 tests from NiRFmxLTESessionTest (5949 ms total)

[----------] Global test environment tear-down
[==========] 7 tests from 2 test suites ran. (43633 ms total)
[  PASSED  ] 7 tests.
```

### CODE REVIEW

File: generated/nirfmxlte/nirfmxlte.proto [hunk 1]
```diff
@@ -13,43 +13,5072 @@ option csharp_namespace = "NationalInstruments.Grpc.NiRFmxLTE";
...
+  NIRFMXLTE_ATTRIBUTE_LINK_DIRECTION = 3145769;
+  NIRFMXLTE_ATTRIBUTE_SPECIAL_SUBFRAME_CONFIGURATION = 3145770;
+  NIRFMXLTE_ATTRIBUTE_NUMBER_OF_DUT_ANTENNAS = 3145771;
+  NIRFMXLTE_ATTRIBUTE_TRANSMIT_ANTENNA_TO_ANALYZE = 3145772;
+  NIRFMXLTE_ATTRIBUTE_SRS_ENABLED = 3145773;
+  NIRFMXLTE_ATTRIBUTE_SRS_SUBFRAME_CONFIGURATION = 3145774;
+  NIRFMXLTE_ATTRIBUTE_SRS_C_SRS = 3145775;
+  NIRFMXLTE_ATTRIBUTE_SRS_B_SRS = 3145776;
+  NIRFMXLTE_ATTRIBUTE_SRS_I_SRS = 3145777;
+  NIRFMXLTE_ATTRIBUTE_SRS_n_RRC = 3145778;
```
Comment (astarche) [inline generated/nirfmxlte/nirfmxlte.proto]: These mixed case attributes snuck through. We need `UPPER_SNAKE` for these per [proto style guidelines](https://developers.google.com/protocol-buffers/docs/style#enums).

File: generated/nirfmxlte/nirfmxlte.proto [hunk 2]
```diff
@@ -13,43 +13,5072 @@ option csharp_namespace = "NationalInstruments.Grpc.NiRFmxLTE";
...
+}
+
+enum DownlinkChannelConfigurationMode {
+  DOWNLINK_CHANNEL_CONFIGURATION_MODE_UNSPECIFIED = 0;
+  DOWNLINK_CHANNEL_CONFIGURATION_MODE_USER_DEFINED = 1;
+  DOWNLINK_CHANNEL_CONFIGURATION_MODE_TEST_MODEL = 2;
+}
+
+enum DownlinkTestModel {
+  DOWNLINK_TEST_MODEL_TM1_1 = 0;
```
Comment (astarche) [inline generated/nirfmxlte/nirfmxlte.proto]: This is a crazy tokenization scheme, but I guess this is correct?

**Review by astarche — APPROVED** [review]

File: generated/nirfmxlte/nirfmxlte.proto (hunk 2 above)
Comment (reckenro) [reply generated/nirfmxlte/nirfmxlte.proto:850]: Yeah... this matches the corresponding enum value names in the .NET API. Looks odd but it's consistent. FWIW I think the underscores correspond to "."s in the E_UTRA test model name they're referring to. "Test model 3.1a" => `TEST_MODEL_3_1A`

**Review by reckenro — APPROVED** [review]

File: generated/nirfmxlte/nirfmxlte.proto (hunk 2 above)
Comment (DavidCurtiss) [reply generated/nirfmxlte/nirfmxlte.proto:850]: Looks to be correct, yeah. In the LV API, those underscores amidst the numbers are periods, e.g. "Test Model 1.1", "Test Model 3.2b", etc.

**Review by mgumble — APPROVED** [review]

**Pull request merged**